example/methods_directory/regression.json

```diff
@@ -48,11 +50,6 @@
             }
         ]
     },
-    "precision": {
-        "perProperty": {
-            "...": "include(regression/precision.json)"
-        }
-    },
```

Comment:
`precision` should keep on being separate from data. It is calculated based on the data in the first place.